Source/Particles/ElementaryProcess/QEDPhotonEmission.H

```diff
@@ -96,6 +96,17 @@ public:
     * @param[in] opt_depth_functor functor to re-initialize the optical depth of the source particles
     * @param[in] opt_depth_runtime_comp index of the optical depth component of the source species
     * @param[in] emission_functor functor to generate photons and update momentum of the source particles
+    * @param[in] a_pti particle iterator to iterate over electrons or positrons undergoing QED photon emission process
+    * @param[in] lev   the mesh-refinement level
+    * @param[in] ngE   ...?
+    * @param[in] exfab constant reference to the FArrayBox of the x component of the electric field
+    * @param[in] eyfab constant reference to the FArrayBox of the y component of the electric field
+    * @param[in] ezfab constant reference to the FArrayBox of the z component of the electric field
+    * @param[in] bxfab constant reference to the FArrayBox of the x component of the magnetic field
+    * @param[in] byfab constant reference to the FArrayBox of the y component of the magnetic field
+    * @param[in] bzfab constant reference to the FArrayBox of the z component of the magnetic field
+    * @param[in] v_galilean ...?
+    * @param[in] a_offset ...?
```

Comment:
@EZoni or @lucafedeli88 maybe? :)
```suggestion
    * @param[in] v_galilean ...?
    * @param[in] a_offset ...?
```

Reply:
Same as before. Not sure about `a_offset`, but for `v_galilean` we could write something like this:
```suggestion
    * @param[in] v_galilean vector components of the Galilean velocity (used with the Galilean PSATD solver)
    * @param[in] a_offset ...?
```